build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 apply plugin: 'eclipse'
 
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
```

Comment:
Switching to Java 7 isn't a change that we can make lightly. This would need to happen as part of a major version bump.